### DISCUSSION TITLE
Cx patcher

### DISCUSSION THREAD

**Discussion by trace7087** [italomandara/CXPatcher#78] [General]
![image](https://github.com/italomandara/CXPatcher/assets/83868440/a2d787e0-c505-4b0d-8e16-d9c957b5598f)
The continue doesn’t allow me to press it to confirm 😔

**Comment by italomandara** [comment]
it's "I will not" and not "i will not" and "Codeweavers", not "codeweavers" get your capitals right and it will work.

**Reply by trace7087** [reply]
Thank you, I didn’t think that was a problem 😬🤷‍♀️
